components/InputBox/index.tsx

```diff
@@ -656,6 +671,38 @@ const InputBox = ({
       }
     }
   };
+
+  // this function is for editing a conversation
+  const onEdit = async () => {
+    let selectedMessage = selectedMessages[0];
+    let conversationId = selectedMessage?.id;
+    let previousMsg = selectedMessage;
```

Comment:
Use proper variable name, it should be `previousConversation`. we should use conversation in code and message.